browser/tasks/gulp-samples.js

```diff
@@ -28,7 +28,7 @@ log('loaded');
 // NOTE you can comment out strings in this array to run subset of samples
 var sampleSource = [
     // including samples for all components:
-    igConfig.SampleCopyPath + '/**/package.json',
+    //igConfig.SampleCopyPath + '/**/package.json',
```

Comment:
PS: Pretty sure you're not supposed to commit this bit :)

Reply:
fixed